plugins/hu.bme.mit.massif.simulink.api/src/hu/bme/mit/massif/simulink/api/Importer.java

```diff
@@ -1187,12 +1200,12 @@ private void createAndAddPort(Block parent, PortProvider portProvider, Handle po
         Port port;
         // State is a special outport kind
         if ("outport".equalsIgnoreCase(portType) || "state".equalsIgnoreCase(portType)) {
-            port = portAdapter.createPort(parent, portHandle, outPorts);
+            port = portAdapter.createPort(this, parent, portHandle, outPorts);
```

Comment:
I would prefer passing specific information or a wrapper DTO instead of the importer itself.